app/src/app/components/contents/runner/utils/parseParameterValue.ts

```diff
@@ -0,0 +1,14 @@
+export const parseParameterValue = (value: string): string | number | boolean | null => {
+  switch (true) {
+    case value === "":
+      return null;
```

Comment:
Wait I am a bit confused, the form won't submit if any of the values are empty right? Why do we have this case?

Reply:
you are right and this was fixed as part of next PR #116